roles/keystone/templates/etc/keystone/uwsgi/keystone-admin.ini

```diff
@@ -0,0 +1,15 @@
+[uwsgi]
+master = true
+processes = 1
+chmod-socket = 666
+
+home = /opt/openstack/current/keystone
+socket = /run/uwsgi/keystone-admin.socket
+pidfile = /run/uwsgi/keystone-admin.pid
+http = 0.0.0.0:35358
```

Comment:
This should probably pull from the `endpoints` role info for the port to run on.